app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,333 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
+{
+
+    /**
+     * @var $model
+     */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 138</br>The first parameter of a multi-line function declaration must be on the line after the opening bracket